tests/unit/deploy/test_deployer.py

```diff
@@ -607,3 +609,284 @@ def badview():
 
     with pytest.raises(ValueError):
         validate_routes(sample_app.routes)
+
+
+class TestLambdaInitialDeploymentWithConfigurations(object):
+    @fixture(autouse=True)
```

Comment:
Why is autouse being used here?  Seems like you can just use the standard `unittest.TestCase` here right?